src/selfie_launch/launch/all_debug.launch

```diff
@@ -1,42 +1,23 @@
 <launch>
-  <!-- COMMON PARAMS -->
+  <!-- PARAMS FROM YAML -->
+  <arg name="file_name" default="$(find selfie_launch)/launch/parameters.yaml"/>
+  <rosparam command="load" file="$(arg file_name)"/>
+
+  <!-- LAUNCH DEPENDENT PARAMS -->
   <arg name="lookahead" default="0.5" />
```

Comment:
Czemu te parametry tutaj zostały? Nie chcemy znowu ich zmieniać w każdym pliku .launch

Reply:
Te parametry zostawiłem ze względu na to, że plik z parametrami miał być includowany do każdego z 3 plików launch (debug, debug_rviz i race). Po prześledzeniu stanu tych plików z developa były to jedyne parametry, które różniły wszystkie 3 launche od siebie. Uznałem, że jeżeli te różnice tutaj były to znaczy, że mają zostać np. prędkość w trybie debug różna niż w trybie race. Jeżeli tak nie jest to usunę, ale jakbyś mógł to odpowiedz jeszcze na ten komentarz - wtedy będę pewny w 100%.

Reply:
Były różne ze wzgledu na to ze jeździłem na jednym launchu i zapomniałem zmienić tego parametru w innych - stad pomysł na ten jeden config z parametrami :)

Reply:
Jedyne co ma zostać w tych launchach to parametry do wizualizacji ktore definiują te launche oraz ścieżki do plików które nie będą się zmieniać

Reply:
A które wartości zostawić? Z launcha race czy debug?